frontend/proto/ml_metadata/README.md

```diff
@@ -0,0 +1,5 @@
+This directory contains protos that were manually downloaded from:
+
+[https://github.com/google/ml-metadata/tree/master/ml_metadata/proto](https://github.com/google/ml-metadata/tree/master/ml_metadata/proto)
+
+TODO: What version of the protos is the server using and how can we stay in sync? 
```

Comment:
nit: we've automated these, see `npm run build:replace`.
consider remove this line and refer to frontend/README for upgrade instructions

Reply:
Sounds good, updated.